### DISCUSSION TITLE
LDAP and/or Open ID Connect (OIDC) Authentication

### DISCUSSION THREAD

**Discussion by tbelway** [Freika/dawarich#385] [Ideas]
It would be awesome if LDAP and/or OIDC authentication (such as keycloak) could be implemented for central user management. I don't see any note of this in issues nor discussion so bringing it up here. It is very similar to [THIS FEATURE REQUEST](https://github.com/Freika/dawarich/discussions/66), except I'm adding LDAP to it. If you feel this is too similar and LDAP should instead be mentioned in the other discussion, feel free to close it, but I feel like it's different enough in implementation design that it should be a competitive alternative.

**Comment by tabacha** [comment]
If https://github.com/Freika/dawarich/discussions/66 is implemented you can configure LDAP there.

**Reply by Freika** [reply]
It's not implemented yet
